core/designsystem/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/designsystem/preview/Annotations.kt

```diff
@@ -1,34 +1,84 @@
 package io.github.droidkaigi.confsched2023.designsystem.preview
 
 import android.content.res.Configuration
+import androidx.annotation.VisibleForTesting
 import androidx.compose.ui.tooling.preview.Preview
+import kotlin.annotation.AnnotationRetention.SOURCE
+import kotlin.annotation.AnnotationTarget.CLASS
+import kotlin.annotation.AnnotationTarget.FUNCTION
+
+/**
+ * Annotate workaround entities for https://github.com/DroidKaigi/conference-app-2023/issues/627
+ */
+@VisibleForTesting
+@Retention(SOURCE)
+@Target(CLASS, FUNCTION)
+annotation class ShowkaseMultiplePreviewsWorkaround
```

Comment:
For removing someday? This seems useful 👀

Reply:
Yes! We should remark code hunks for this workaround cuz they are large and are crosscutting concerns.